liquibase-standard/src/main/java/liquibase/datatype/core/UUIDType.java

```diff
@@ -48,6 +48,17 @@ public DatabaseDataType toDatabaseDataType(Database database) {
                 throw new RuntimeException(ex);
             }
         }
+        // Not sure if version specific yet
+//        try {
+//            if (database instanceof MySQLDatabase && (database.getDatabaseMajorVersion() == 8)) {
```

Comment:
As per the [blog mentioned below](https://dev.mysql.com/blog-archive/mysql-8-0-uuid-support/) , uuid was introduced in Mysql 8
For MariaDB it was introduced[ in version 10.7 ](https://mariadb.com/kb/en/uuid-data-type/), so this PR could also support MariaDB.

Reply:
I think MariaDB is already implemented above on line 40. Is that what you meant?